src/lotus/message.rs

```diff
@@ -72,11 +71,10 @@ pub struct ReadStateResponse<State> {
 #[serde(rename_all = "PascalCase")]
 pub struct Receipt {
     #[allow(dead_code)]
-    exit_code: u32,
+    pub exit_code: u32,
+    pub r#return: String,
```

Comment:
I know that this is not your idea/fault and it comes from Lotus, but man this is horrible haha

Reply:
You mean the `return` right? We can rename this to `return_string` if we want to avoid `r#`.

Reply:
yeah, I changed to `result` instead of `return`, maybe it's better than `return`.

Reply:
In the end we needed a renaming. Resolving this.